commands/utils/git.go

```diff
@@ -254,3 +312,16 @@ func toBasicAuth(token, username string) *http.BasicAuth {
 func getFullBranchName(branchName string) plumbing.ReferenceName {
 	return plumbing.NewBranchReferenceName(plumbing.ReferenceName(branchName).Short())
 }
+
+func loadCustomFormats(formatsArray map[string]string) (CustomFormats, error) {
```

Comment:
formatsArray --> formats
(this is not an array, but a map, so you cannot call this variable an array).

This function should also be unit tested. The error it returns in case of `IsValidBranchFormat` should also be tested.